familias/utils.py

```diff
@@ -0,0 +1,47 @@
+import decimal
+from django.shortcuts import get_object_or_404
+from indicadores.models import Transaccion
+from .models import Familia
+
+
+def total_egresos_familia(id_familia):
+    """ Returns the total value of the losses a
+    family has per month.
+
+    """
+    familia = get_object_or_404(Familia, pk=id_familia)
+    transacciones = Transaccion.objects.filter(familia=familia,
+                                               activo=True,
+                                               es_ingreso=False)
+    total = decimal.Decimal('0')
+    for transaccion in transacciones:
+        total = total + transaccion.obtener_valor_mensual()
+
+    return total;
+
+def total_ingresos_familia(id_familia):
```

Comment:
Also it isn't necessary but I would just create one function which receives the es_ingreso, and depending on its value make the query for the transactions (if it is None, you don't pass it, for total_neto). And the total can be a one liner using sum().